reference/twig-extensions/functions/sulu_navigation_flat.rst

```diff
@@ -7,7 +7,7 @@ Returns navigation from the given page in a flat list data-structure.
 
 - **uuid**: *string* The uuid for which the navigation should be loaded
 - **context**: *string* - optional: context to filter navigation
-- **depth**: *integer* - optional: depth to load (1 - childs, 2 - childs and child of childs, ...)
+- **depth**: *integer* - optional: depth to load (1 - children, 2 - children and descendants, ...)
```

Comment:
`descendants` is not really true here... This will only return its linear descendants, not e.g. their grand children. But instead of using harder language I would suggest to `children and their children`.